build/Linux+BSD/portable/RecipeArm

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+# This file is designed to run in debian jessie docker images
+
+main() {
+
+  set -e # Halt on error
+  set -x # Be verbose
+
+  case $1 in
+    "--fetch-dependencies-only")
+      fetchDependencies "$@"
+      ;;
+    "--build-only")
+      shift #need to adjust parameter list so $makefile_overrides start at $1
+      build "$@"
+      ;;
+    "--package-only")
+      package "$@"
+      ;;
+    *)
```

Comment:
Looks fine to me, but you want another option `--fetch-dependencies-only` that would run a new function "fetch_dependencies". Move all of the apt-get lines out of your dockerfile and put them in this function, then do a `wget` and `--fetch-dependencies-only` in the dockerfile. The only thing you should `apt-get` in the dockerfile is `wget` itself.